src/react/account/CreateAccountPolicy.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Box, Button } from '@scality/core-ui/dist/next';
+import { useForm } from 'react-hook-form';
+import FormContainer, * as F from '../ui-elements/FormLayout';
+import { SecondaryText  } from '@scality/core-ui/dist/components/text/Text.component';
+import CopyButton from '../ui-elements/CopyButton';
+import { useMutation  } from 'react-query';
+import { getListPoliciesQuery } from '../queries';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useIAMClient } from '../IAMProvider';
+import { queryClient } from '../App';
+import { useHistory, useParams } from 'react-router-dom';
+
+type FormValues = {
+  policyName: string;
+  policyDocument: string;
+};
+
+type Props = {
+  accountName: string;
+  policyName: string;
+  policyDocument: string;
+}
+
+const CreateAccountPolicy = () => {
+  const IAMClient = useIAMClient();
+  const history = useHistory();
+  const { accountName } = useParams();
+
+  const defaultValues = {
+    policyName: '',
+    policyDocument: '',
+  };
+  const {
+    register,
+    reset,
+    handleSubmit,
+    watch,
+    formState: { isDirty, errors,  },
+  } = useForm<FormValues>({
+    defaultValues
+  });
+
+  const watchAllFields = watch();
+
+  const createPolicyMutation = useMutation(
+    ({ policyName, policyDocument } : Props) => {
+      return notFalsyTypeGuard(IAMClient).createPolicy(
+        policyName,
+        policyDocument,
+      );
+    },
+    {
+      onSuccess: () =>
+        queryClient.invalidateQueries(
+          getListPoliciesQuery(accountName, notFalsyTypeGuard(IAMClient))
+            .queryKey,
+        ),
+    },
+  );
+
+  const onSubmit = (data: FormValues) => {
+    const policyName = data.policyName;
+    const policyDocument = data.policyDocument;
+    createPolicyMutation.mutate({ policyName, policyDocument });
+    reset(defaultValues);
+  };
+
+  const handleCancel = (e) => {
+    if (e) {
+      e.preventDefault();
+    }
+    history.goBack();
+  };
+
+  return (
+    <FormContainer>
+      <F.Form onSubmit={handleSubmit(onSubmit)}>
+        <F.Title>Policy Creation</F.Title>
+        <F.Hr/>
+        <SecondaryText> All * are mandatory fields </SecondaryText>
+          <F.Fieldset>
+          <F.Label
+            htmlFor="policyName"
+            tooltipMessages={['The policy name is required']}
+            tooltipWidth="10.5rem"
```

Comment:
We could pass the required prop to the F.Label instead of adding the *
```suggestion
          <F.Label
            htmlFor="policyName"
            tooltipMessages={['The policy name is required']}
            tooltipWidth="10.5rem"
            required
```